utils/linux-install-luarocks.sh

```diff
@@ -41,7 +43,6 @@ fi
 make build > build.log 2>&1 || (cat build.log && exit 1)
 sudo make install > build.log 2>&1 || (cat build.log && exit 1)
 cd .. || exit
-rm -rf luarocks-3.4.0
```

Comment:
Why was this cleanup removed?

Reply:
My mistake, I will add the cleanup code.